coordinator/src/orderbook/trading.rs

```diff
@@ -152,7 +152,7 @@ fn sort_orders(mut orders: Vec<Order>, is_long: bool) -> Vec<Order> {
 #[autometrics]
 pub async fn notify_traders(
     matched_orders: MatchParams,
-    authenticated_users: HashMap<PublicKey, Sender<OrderbookMsg>>,
+    authenticated_users: &HashMap<PublicKey, Sender<OrderbookMsg>>,
```

Comment:
❓ My understanding is, that now this function call just takes a reference, i.e. not consuming the map. 
Is this a pure refactor or does it actually make a difference in terms of blocking / mutex?